### PR TITLE
feat: DAH-2238: Account Dashboard

### PR DESCRIPTION
**Pull request by chadbrokaw**
[DAH-2338](https://sfgovdt.jira.com/browse/DAH-2338)

[DAH-2338]: https://sfgovdt.jira.com/browse/DAH-2338?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by tallulahkay — CHANGES_REQUESTED** [review]
This looks good! Left one non-blocking comment, and will approve when the redirect works as expected (left a comment on the jira ticket)

File: app/javascript/pages/account/my-account.tsx [hunk 1]
```diff
@@ -1,15 +1,89 @@
...
+  )
+}
+
 const MyAccount = (_props: MyAccountProps) => {
+  const { profile, loading, initialStateLoaded } = React.useContext(UserContext)
+
+  if (!profile && !loading && initialStateLoaded) {
+    // TODO: Redirect to React sign in page and show a message that user needs to sign in
+    window.location.href = getSignInPath()
+    return null
```
Comment (cade-exygy) [inline app/javascript/pages/account/my-account.tsx:62]: Does setting the `window.location.href` execute the redirect or do we need to do something in addition here?

**Review by cade-exygy — APPROVED** [review]
Except for the redirect error, testing criteria passes. Tested in PR environment.

Good to merge once redirect error is fixed (or validating it works correctly)

**Comment by cade-exygy** [comment]
> Except for the redirect error, testing criteria passes. Tested in PR environment.
> 
> Good to merge once redirect error is fixed (or validating it works correctly)

Just saw updated instructions. Testing with `my-account?react=true` works!

**Review by tallulahkay — APPROVED** [review]
Looks good!!

**Pull request merged**